### DISCUSSION TITLE
Zoom Meeetings Showing as "Inactive"

### DISCUSSION THREAD

**Discussion by jbearden41** [code4recovery/12-step-meeting-list#825] [Q&A] [answered]
I.m using the 12-Step Meeting List plugin with my (experimental. at this stage) WordPress Web site (test.cnia30.org), and following all the instructions (I think) for entering meeting information into the template spreadsheet (CSV file) provided by them. As recommended, after it refused to import any meeting that didn't have both a meeting location and address field, I filled in both of these (as also recommended) with just the town and state the meeting was associated with. The Zoom IDs and passwords were put into the "Conference URL" and "Conference URL Notes", fields, respectively. This allowed the meeting information to be imported, but all of the on-line meetings were shown as "Inactive", and of course, no Zoom login information was shown. Can anyone tell me what is still going wrong with this, and what I need to do in order to get these meetings properly listed? Thanks for any help you can offer.

**Comment by joshreisner** [comment, marked as answer]
hi! could you share the CSV so we can see what's going on? a meeting will display as inactive if it doesn't have a specific street address and no recognizable conference_url (or conference_phone)

**Reply by jbearden41** [reply]
Thanks for your help. I've attached the CSV template file. as I filled it out, so possibly you can see what I might still be doing wrong.
[AllMeetings-12StepTemplate.csv](https://github.com/code4recovery/12-step-meeting-list/files/8873016/AllMeetings-12StepTemplate.csv)
g

**Reply by joshreisner** [reply]
ah, i didn't notice you wrote "zoom ids" above -- that's the issue here. `conference url` takes a full url to one of several services. if you just prepend `https://zoom.us/j/` to each of those ids and re-upload you should be good to go

**Reply by jbearden41** [reply]
Thanks, I'll try that.

**Reply by jbearden41** [reply]
Thanks! That worked. All of the Zoom meetings (including the Hybrids) are now showing up properly. You might add your explanation of how to enter Zoom information properly into the spreadsheet to your "Spreadsheet example and spec" description, for people like me who are too dumb to figure that out. I've got just one remaining problem, and I attached a copy of my revised spreadsheet file to see if you can help me figure out the last problem with just one meeting: the Tuesday noon "Daily Reflections" meeting in Murphys is still showing as "Inactive", even though, as far as I can tell, the address and other information is all valid, as far as Google maps is concerned (and it's not even a Zoom meeting). Any ideas what I might still be doing wrong with that one? Thanks.
[AllMeetings-12StepTemplate.csv](https://github.com/code4recovery/12-step-meeting-list/files/8874170/AllMeetings-12StepTemplate.csv)

**Reply by joshreisner** [reply]
sometimes the google geocoding api doesn't like a particular address, that's what appears to be happening in this case.

try adding this code to your theme's functions.php file (or child theme, or `wp-config.php`):

```
if (function_exists('tsml_custom_addresses')) {
    tsml_custom_addresses(array(
        '480 Park Lane, Murphys, CA 95247' => array(
            'formatted_address' => '480 Park Ln, Murphys, CA 95247, USA',
            'city' => 'Murphys',
            'latitude' => 33.869740,
            'longitude' => -117.560420,
            'approximate' => 'no',
        ),
    ));
}
```

hopefully this will override what google is coming back with

**Reply by jbearden41** [reply]
This did not fix the problem, which appears to be in the "12 Step Meeting List" plugin, NOT in Google (if I enter the address into Google Maps, it finds it with no trouble). When I edit the meeting information, including the address (in Murphys, CA), it forces the location to a "Park Lane" address in Corona (in southern California), and then labels it "Inactive", and it does this no matter how many times I enter the correct (full) address in Murphys. Josh Reisner was helping me with other problems I was having, but this one seems to have him stumped, since I haven't heard how to fix it after 3 weeks of waiting. Does anyone else in the Code4Recovery group know how to fix this bug? Thanks for any help you can offer.

**Reply by tech2serve** [reply]
Hey @jbearden41,

I suspect Josh was waiting to hear back from you after he provided instructions on how to override the address.

And since this is really a separate issue from the OP, please create a new discussion thread (Q&A or general) for the geocoding challenge. 

In it, please include the address as you're typing it so that one of us and try and replicate on a different install of TSML. Screen caps of the correct location (from maps.google.com) can be helpful. 

Unfortunately, it is possible to get different answers from Google depending on which API is used. It doesn't happen very often, but it is possible. Hence why Josh provided the override. If correctly added to your system, Google's servers won't be touched in order to geocode that address. 

So, to track this down we'll need to look closer at the responses In your website. If you can open a javascript console, and then enter the address for geocoding (on the admin edit meeting page), you'll see the `geocode` object (or something similar, I'm doing it from memory right now :-) ). Clicking on that will provide more information about where the address is coming from. 

As I said though, please open a new discussion.

Thanks.
